src/librustdoc/html/static/styles/main.css

```diff
@@ -26,9 +26,6 @@ h1.fqn {
 h2, h3:not(.impl):not(.method):not(.type):not(.tymethod), h4:not(.method):not(.type):not(.tymethod) {
 	border-bottom-color: #DDDDDD;
 }
-.in-band {
-	background-color: white;
-}
```

Comment:
Strange to have `background` and `background-color` and that your CSS change doesn't work...